### PR TITLE
Development

### PR DESCRIPTION
**Pull request by chuanli2**
I'd like to open a pull request,  It is not ready to merge.  I still have to update documentation and make minor tweaks to the code before its ready.  

Please stand by

I've added: 
- ability to change models in get wave spectral data  and field data

- I've added get cshore data

### CODE REVIEW

**Review by SBFRF — CHANGES_REQUESTED** [review]
please update port calls (8080) add docstrings for new functions

**Comment by chuanli2** [comment]
Removed 8080s and added docstring for getCSHOREOutput. Ready to merge.

File: getDataFRF.py
Comment (SBFRF) [inline getDataFRF.py:2444]: Try to do something like this.  I think its a good way to not break things, but will move things forward as well! 

    def getWaveSpecSTWAVE(self, prefix, gaugenumber, local=True, model='STWAVE'):
        warnings.warn('Using depricated function name')
        return self.getWaveSpecSTWAVE(prefix, gaugenumber, local, model)

    def getWaveSpecModel(self, prefix, gaugenumber, local=True, model='STWAVE'):

File: getDataFRF.py
Comment (SBFRF) [inline getDataFRF.py]: : needs to be removed

File: getDataFRF.py
Comment (SBFRF) [inline getDataFRF.py:2499]: :+1:

File: getDataFRF.py
Comment (SBFRF) [inline getDataFRF.py]: please define output dictionary (similar to above functions)

File: getDataFRF.py
Comment (SBFRF) [inline getDataFRF.py]: please remove ':'

**Review by SBFRF — COMMENTED** [review]
Here are a few things i found that would need to be fixed before an approval

File: getDataFRF.py
Comment (SBFRF) [reply getDataFRF.py:2444]: also this would work for the STWAVE field function

Comment (chuanli2) [reply getDataFRF.py:2444]: Should be like this instead right?

return self.getWaveSpecModel(prefix, gaugenumber, local, model)

Comment (chuanli2) [reply getDataFRF.py:2444]: Made the change.

File: getDataFRF.py
Comment (chuanli2) [reply getDataFRF.py]: Done

File: getDataFRF.py
Comment (chuanli2) [reply getDataFRF.py]: Done

File: getDataFRF.py
Comment (chuanli2) [reply getDataFRF.py]: Done

**Comment by chuanli2** [comment]
Made second round of changes. Ready to merge unless there are more change requests.

**Review by SBFRF — APPROVED** [review]
Looks Good!

**Pull request merged**
